client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/RemoteCacheConfigurationBuilder.java

```diff
@@ -166,6 +166,7 @@ public RemoteCacheConfigurationBuilder templateName(String templateName) {
     * @param template the template to use
     * @return an instance of the builder
     */
+   @Deprecated(forRemoval = true)
```

Comment:
Maybe update the javadoc with the alternative?

Reply:
Done